Description: add git commit hash again
Author: Elliana May <me@mause.me>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2024-03-01

--- duckdb-0.10.0.orig/CMakeLists.txt
+++ duckdb-0.10.0/CMakeLists.txt
@@ -303,6 +303,7 @@ set(DUCKDB_NORMALIZED_VERSION "v0.10.0")
 set(DUCKDB_VERSION_MAJOR 0)
 set(DUCKDB_VERSION_MINOR 10)
 set(DUCKDB_VERSION_PATCH 0)
+set(GIT_COMMIT_HASH "20b1486d11")
 
 if(EMSCRIPTEN)
   set(EXTENSION_POSTFIX "duckdb_extension.wasm")
